### PR TITLE
Anomalous Crystal: We hate clowns

### PR DESCRIPTION
**Pull request by Therandomhoboo**
# Document the changes in your pull request

We hate clowns, clowns are dumb, clumsy and goofy and we as a NRP server, we hate them. Fighting a colossus and potentially being rewarded a forced job change whilst it makes you drop all your items isn't fun at all. I am leaving this up to the community and personally don't care what happens. 

# Why is this good for the game?

Stops people from being clowned because they beat a boss that is apparently very hard, kinda lame to be rewarded with a slap in the face and a "job" change whilst dropping all your shit.

Also forgot to mention, stops miners or griefers from abusing said crystal effect to ruin people's rounds.

# Wiki Documentation

The HONK and anything clown related needs to be removed from the wiki and redacted so no one knows what we did.

# Changelog

:cl:  

rscdel: Removed the potential to spawn the clown effect for the anomalous crystal after you beat the colossus megafauna

/:cl:

### CODE REVIEW

**Comment by ofscarletdreams** [comment]
maybe reword the changelog to removed the clown effect from the anomalous crystal because it sounds like you straight up removed the crystal itself

**Comment by Therandomhoboo** [comment]
> maybe reword the changelog to removed the clown effect from the anomalous crystal because it sounds like you straight up removed the crystal itself

Done chief

**Comment by ofscarletdreams** [comment]
:)

**Review by SapphicOverload — APPROVED** [review]
code looks fine

**Comment by neeshacark** [comment]
i thought we already removed this option

yeah this thing sucks, it literally fucks miners almost entirely because now they cant use their cutters to mine and i think cant use their pka either.  this means they cant mine which sucks for and sucks for crew because now there's less mats coming in.
